sdk/digitaltwins/azure-digitaltwins-core/pom.xml

```diff
@@ -78,6 +78,11 @@
   </dependencies>
 
   <build>
+    <resources>
+      <resource>
+        <directory>src/samples/resources</directory>
```

Comment:
maven has inbuilt references to src/main/resources and src/test/resources. We need to add reference to src/samples/resources explicitly.